src/rules/no-entire-object-hook-deps.ts

```
@@ -295,10 +314,79 @@ function getObjectUsagesInHook(
+          }
+        }
+      }
+    } else if (node.type === AST_NODE_TYPES.ChainExpression) {
+      // Handle optional chaining expressions
+      if (node.expression.type === AST_NODE_TYPES.MemberExpression) {
+        const path = buildAccessPath(node.expression);
+        if (path) {
+          usages.set(path, node.range?.[0] || 0);
+        }
       }
+    } else if (node.type === AST_NODE_TYPES.BinaryExpression || node.type === AST_NODE_TYPES.LogicalExpression) {
```

ESLint

[error] Replace `node.type·===·AST_NODE_TYPES.BinaryExpression·||·node.type·===·AST_NODE_TYPES.LogicalExpression` with `⏎······node.type·===·AST_NODE_TYPES.BinaryExpression·||⏎······node.type·===·AST_NODE_TYPES.LogicalExpression⏎····`

(prettier/prettier)